oaknational/Oak-Web-Application
# chore: address a bunch of console warnings and a11y concerns

## Description

- adds aria label to navigation on about page
- fixes the console errors from [TransitionGroup in strict mode](https://github.com/reactjs/react-transition-group/pull/851)
- changes about: contact section heading tags so they don't jump level
- changes about: document heading tags so they don't jump level
- changes about/board "board members" to be a list rather than headings (which jumped in level)

## How to test

1. Go to {about page}
2. Check axe (or wave)
3. You shoudn't see a11y warnings about levels or navigation without label


## Checklist
